src/main/java/org/assertj/core/internal/Paths.java

```diff
@@ -496,4 +499,22 @@ private static void assertExpectedEndPathIsNotNull(final Path end) {
     requireNonNull(end, "the expected end path should not be null");
   }
 
+  public void assertIsEmptyPath(AssertionInfo info, Path actual) {
+    assertIsRegularFile(info, actual);
+    try {
+      if (size(actual) == 0) return;
```

Comment:
I would invert the logic here:
```suggestion
      if (size(actual) > 0) throw failures.failure(info, shouldBeEmpty(actual));
```

I suggest adding `public static ErrorMessageFactory shouldBeEmpty(Path actual)` to `ShouldBeEmpty`.